src/services/AuthService.ts

```diff
@@ -1,4 +1,4 @@
-import { Auth } from 'aws-amplify';
+import { Auth } from "aws-amplify";
```

Comment:
I suppose that you need to add to add [double quotes](https://eslint.org/docs/latest/rules/quotes) rule to eslint config